configs/crypto-config-ccm-aes-sha256.h

```diff
@@ -24,5 +24,6 @@
 #define MBEDTLS_PSA_CRYPTO_C
 #define MBEDTLS_CTR_DRBG_C
 #define MBEDTLS_ENTROPY_C
+#define MBEDTLS_PLATFORM_C
```

Comment:
This shouldn't be necessary. `MBEDTLS_PLATFORM_C` should only be needed to support runtime configuration of platform features. Neither `MBEDTLS_PLATFORM_GET_ENTROPY_ALT` nor `!MBEDTLS_PLATFORM_GET_ENTROPY_ALT` should require `MBEDTLS_PLATFORM_C`.

I think the natural way to fix this is to move the code from `platform.c` to `platform_util.c`.

At this point, given that we're planning to do a disruptive combined merge tonight, I propose to merge as is, and do a follow-up that does the move and removes the added enabling of `MBEDTLS_PLATFORM_C` in both TF-PSA-Crypto and mbedtls.

Reply:
https://github.com/Mbed-TLS/TF-PSA-Crypto/issues/282